django/test/testcases.py

```diff
@@ -140,6 +140,19 @@ def message(self):
         return '%s was rendered.' % self.template_name
 
 
+class _CursorFailure(object):
+    def __init__(self, cls_name, wrapped):
+        self.cls_name = cls_name
+        self.wrapped = wrapped
+
+    def __call__(self):
+        raise AssertionError(
+            "SimpleTestCase shouldn't be used to test features relying on the ORM. "
```

Comment:
perhaps more generally, "Database queries aren't allowed in SimpleTestCase." (since queries can be executed in ways other than just the oRM)